plugins/kontent-plugin/gatsby-node.js

```diff
@@ -0,0 +1,858 @@
+const { getGatsbyImageResolver } = require("gatsby-plugin-image/graphql-utils")
+const { getImageData } = require("gatsby-plugin-image")
+const {
+  getKontentItemNodeTypeName,
+} = require("@kentico/gatsby-source-kontent/src/naming")
+const { createContentDigest } = require("gatsby-core-utils")
+const crypto = require("crypto")
```

Comment:
This is a deprecated package, and as such I would suggest removing it.

Based on it's usage, I think it would be very easy to replace with `uuid`: https://www.npmjs.com/package/uuid